cmd/legacy/run.go

```diff
@@ -114,11 +115,18 @@ func Run(cmd *cobra.Command, v *viper.Viper) {
 		RunCmd(v, offlinesync.Run)
 	}
 
+	if v.GetBool("offline-count") {
+		log.Debugln("command: offline-count")
+
+		RunCmd(v, offlinecount.Run)
+	}
+
 	log.Warnf("One of the following parameters has to be provided: %s", strings.Join([]string{
 		"--config-read",
 		"--config-write",
 		"--entity",
 		"--sync-offline-activity",
+		"--offline-count",
```

Comment:
Just to maintain the alphabetic sorting.
```suggestion
		"--offline-count",
		"--sync-offline-activity",
```